core/src/main/scala/sttp/tapir/ShadowedEndpoint.scala

```diff
@@ -0,0 +1,5 @@
+package sttp.tapir
+
+case class ShadowedEndpoint(e: Endpoint[_, _, _, _], by: Endpoint[_, _, _, _]) {
+  override def toString = e.input.show + " is shadowed by " + by.input.show
```

Comment:
[minor] I find it useful to clearly delimit where data and helper text begins/ends. So I'd do: `e.input.show ", is shadowed by: " + by.input.show`